types/log4js.d.ts

```diff
@@ -272,11 +272,12 @@ export interface Levels {
 
 export interface Configuration {
 	appenders: { [name: string]: Appender; };
-	categories: { [name: string]: { appenders: string[]; level: string; } };
+	categories: { [name: string]: { appenders: string[]; level: string; enableCallStack?: boolean; } };
 	pm2?: boolean;
 	pm2InstanceVar?: string;
 	levels?: Levels;
 	disableClustering?: boolean;
+  enableCallStack?: boolean;
```

Comment:
This one doesn't exist. Only the category level one is used.

Reply:
ok,  It has been modified.